### DISCUSSION TITLE
How to add meta data to a page, for example for SEO?

### DISCUSSION THREAD

**Discussion by Bambus89** [datenstrom/community#274] [Ask a question] [answered]
Hello Everyone...
Is there any option for adding Metadata and a full Description for a Yellow Installation?
Maybe i am blind but i not found anything in Config files...

**Comment by schulle4u** [comment, marked as answer]
Meta description and keywords can be set in the page settings at the top of each content file. Have a look at the [settings section in the markdown cheat sheet](https://developers.datenstrom.se/help/markdown-cheat-sheet#settings). Description and keywords currently have to be set for each page separately.

However, as far as I know search engines nowadays don't care that much about keywords anymore. 

HTH
Steffen

**Comment by markseuffert** [comment]
Additionally to what Steffen said, you can add your own metadata to the header snippet. Description and keywords are already there. For more information how templates and snippets work, see https://datenstrom.se/yellow/help/

**Comment by Bambus89** [comment]
Thank YOu, That works
